skbio/sequence/_genetic_code.py

```diff
@@ -8,588 +8,776 @@
+    def translate(self, sequence, reading_frame=1, start='ignore',
+                  stop='ignore'):
+        """Translate RNA sequence into protein sequence.
 
         Parameters
         ----------
-        codon : str
-            codon string
+        sequence : RNA
+            RNA sequence to translate.
+        reading_frame : {1, 2, 3, -1, -2, -3}
+            Reading frame to use in translation. 1, 2, and 3 are forward frames
+            and -1, -2, and -3 are reverse frames. If reverse (negative), will
+            reverse complement the sequence before translation.
+        start : {'ignore', 'require', 'optional'}
+            How to handle start codons:
+
+            * "ignore": translation will start from the beginning of the
+              reading frame, regardless of the presence of a start codon.
+
+            * "require": translation will start at the first start codon in
+              the reading frame, ignoring all prior positions. The first amino
+              acid in the translated sequence will *always* be methionine
+              (M character), even if an alternative start codon was used in
+              translation (this behavior most closely matches the underlying
+              biology). If a start codon does not exist, a ``ValueError`` is
+              raised.
+
+            * "optional": if a start codon exists in the reading frame, matches
+              the behavior of "require". If a start codon does not exist,
+              matches the behavior of "ignore".
+
+        stop : {'ignore', 'require', 'optional'}
+            How to handle stop codons:
+
+            * "ignore": translation will ignore the presence of stop codons and
+              translate to the end of the reading frame.
+
+            * "require": translation will terminate at the first stop codon.
+              The stop codon will not be included in the translated sequence.
+              If a stop codon does not exist, a ``ValueError`` is raised.
+
+            * "optional": if a stop codon exists in the reading frame, matches
+              the behavior of "require". If a stop codon does not exist,
+              matches the behavior of "ignore".
 
         Returns
         -------
-        bool
-            ``True`` if codon is a stop codon, ``False`` otherwise
+        Protein
+            Translated sequence.
+
+        See Also
+        --------
+        translate_six_frames
+
+        Notes
+        -----
+        Input RNA sequence metadata are included in the translated protein
+        sequence. Positional metadata are not included.
 
         Examples
         --------
-        >>> from skbio.sequence import GeneticCode
-        >>> sgc = GeneticCode('FFLLSSSSYY**CC*WLLLLPPPPHHQQRRRRIIIMTTTTNNKKSS'
-        ...                   'RRVVVVAAAADDEEGGGG')
-        >>> sgc.is_stop('UAA')
-        True
-        >>> sgc.is_stop('AAA')
-        False
+        Translate RNA into protein using NCBI's standard genetic code (table ID
```

Comment:
Since this is the main translate documentation, I wonder if it's worth noting what this genetic code is here (i.e., that it's the *human nuclear genetic code*).